hoomd/md/PotentialTersoffGPU.cuh

```diff
@@ -118,7 +118,7 @@ __device__ double myAtomicAdd(double* address, double val)
 
 // workaround for HIP bug
 #ifdef __HIP_PLATFORM_HCC__
-inline __device__ float myAtomicAdd(float* address, float val)
+inline __device__ inline float myAtomicAdd(float* address, float val)
```

Comment:
```suggestion
__device__ inline float myAtomicAdd(float* address, float val)
```

Reply:
Without the inline, you get multiply defined symbol errors when the header is included in multiple compilation units.

Reply:
Sure, we can change the order. The compiler will accept these in either order.

Reply:
Inline was being declared twice here. Was that needed?

Reply:
Sorry, I misread the diff. One inline is enough. Not sure why I added a 2nd except that I was getting a compile error with this at one point while developing this branch.